grafana-plugin/src/pages/incident/Incident.tsx

```diff
@@ -177,13 +177,15 @@ class IncidentPage extends React.Component<IncidentPageProps, IncidentPageState>
                   </div>
                   <div className={cx('column')}>
                     <VerticalGroup style={{ display: 'block' }}>
-                      <AddResponders
-                        mode="update"
-                        hideAddResponderButton={incident.resolved}
-                        existingPagedUsers={incident.paged_users}
-                        onAddNewParticipant={this.handleAddUserResponder}
-                        generateRemovePreviouslyPagedUserCallback={this.handlePagedUserRemove}
-                      />
+                      {(!incident.resolved || incident.paged_users.length > 0) && (
+                        <AddResponders
+                          mode="update"
+                          hideAddResponderButton={incident.resolved}
+                          existingPagedUsers={incident.paged_users}
+                          onAddNewParticipant={this.handleAddUserResponder}
+                          generateRemovePreviouslyPagedUserCallback={this.handlePagedUserRemove}
+                        />
+                      )}
```

Comment:
don't show the "Participants" section if the alert group has been resolved and it has no `paged_users` associated with it